detection-rules/spam_attendee_list_solicitation.yml

```diff
@@ -0,0 +1,28 @@
+name: "Spam: Attendee List solicitation"
+description: "This rule detects messages claiming to have the attendee list from a specific event, they may list various information such as the number of contacts, the demographic and sample contacts. The messages typically offer to send pricing information upon request."
+type: "rule"
+severity: "low"
+source: |
+  type.inbound
+  and length(body.current_thread.text) < 2000
+  and length(body.links) < 5
+  and any([body.html.display_text, body.current_thread.text],
+          regex.icontains(body.current_thread.text,
+                          "(Attendee|Member|Participants) (list|database)"
+          )
+  )
```

Comment:
I'm a little confused about this part, is it supposed to be `regex.icontains(.`?
Also, why both `body.html.display_text` and `body.current_thread.text`?

Reply:
It is supposed to be icontains, I just didn't lower the strings since it doesn't matter anyway :) 
The aggregation of display_text and body.current_thread.text is both an artifact from joshes original rule, but also because in testing without it, we miss some. :/ 

![image](https://github.com/sublime-security/sublime-rules/assets/6809735/a92732bb-1cfe-4f6d-b197-2f074cd4e6bf)